src/ray/raylet/worker_pool.cc

```diff
@@ -153,14 +153,14 @@ Process WorkerPool::StartWorkerProcess(
   // Here we consider both task workers and I/O workers.
   if (starting_workers >= maximum_startup_concurrency_) {
     // Workers have been started, but not registered. Force start disabled -- returning.
-    RAY_LOG(DEBUG) << "Worker not started, " << starting_workers
-                   << " workers of language type " << static_cast<int>(language)
-                   << " pending registration";
+    RAY_LOG(WARNING) << "Worker not started, " << starting_workers
```

Comment:
This also shouldn't be warning. There's no way any user will know what to do after looking at this from their driver.